src/Domain/ReadableIdGenerator.cs

```diff
@@ -4,7 +4,7 @@ namespace UltimateTicTacToe.Domain
 {
     public static class ReadableIdGenerator
     {
-        private const string Characters = "ABCDEFGHIJKLMNOPQRSTUVQXYZabcdefghijklmnopqrstuvwxyz";
+        private const string Characters = "ABCDEFGHIJKLMNOPQRSTUVQXYZabcdefghijklmnopqrstuvwxyz123456789";
```

Comment:
0 is missing but probably better to omit it because O and 0 look way too similar.